docs/guides/insar_product_guide.md

```diff
@@ -344,6 +349,15 @@ When looking at a single interferogram, the only reliable deformation measuremen
 
 In addition, a single interferogram cannot be used to determine the relative contributions of vertical and horizontal movement to the line-of-sight displacement measurement. The vertical displacement map is generated based on the assumption that the movement is entirely in the vertical direction, which may not be realistic for some processes. To determine how much of the signal is driven by vertical vs. horizontal movement, you must either use a time series of interferograms, or use reference measurements with known vertical and horizontal components (such as GNSS measurements from the region of deformation) to deconstruct the line-of-sight displacement.
 
+### Phase Unwrapping Reference Point
+The reference point for phase unwrapping is set to be the upper left corner of the image by default. This may not always be an ideal location. If it's in an area of low coherence, or if it's in a patch of coherent pixels that is separated from the area undergoing deformation by a gap of incoherent pixels, the unwrapping may be of lower quality than if the reference point was in a more suitable location. 
+
+Even when there are not phase unwrapping errors introduced by phase discontinuities, it is important to be aware that unwrapped phase differences and displacement values are all relative to the reference point. If you are interested in the absolute change in displacement in a particular area, you may want to apply a correction to the image so that the values you are working with are in terms that are more intuitive for your study area or application.
+
+In cases where there are not significant phase discontinuities, you can select a reference point that is optimal for your use case, calculate the difference in phase or line-of-sight displacement, and apply a correction to the entire raster to adjust the values to be relative to that new reference point.
```

Comment:
I think this needs more clarification - perhaps an example, like 

_If the reference point at 0,0 has a phase of 0.3, and your desired reference point has a phase of 0.7, you would subtract 0.7 from all of the phase values in your image, changing the initial reference point to a value of -.4._

Reply:
I added an example box for this [in this commit](https://github.com/ASFHyP3/hyp3-docs/pull/158/commits/fd1a78bf9126c7201633b64c3d145380340c3ad2). Might be too wordy. Thought I might add an image to help illustrate. Will follow up next week. Comments welcome.

Reply:
The story was a bit complicated when adding images. At @talogan's suggestion, I just took the first sentence from the example box and added it to the previous paragraph (see [this commit](https://github.com/ASFHyP3/hyp3-docs/pull/158/commits/c86484d957abfd24bc0d05698f2a4ea525ac2935). Someday I hope to put together a story map that dives into this more.